### ISSUE TITLE
MAGFest Volunteer Badge Confirmation email doesn't specify which event

### ISSUE THREAD

**Issue by kitsuta** [magfest/ubersystem#1494]
Per a conversation with Rene and Carol, many of their volunteers received the "MAGFest Volunteer Badge Confirmation" email for MAGFest 2016 but assumed it was for MAGClassic. The email doesn't specify which event - either in the subject line or the body of the email - so this is confusing to volunteers.

**Comment by kitsuta** [comment]
Having checked the code, it does use the EVENT_NAME variable, so there's not much to be done code-wise here.

**Comment by binary1230** [comment]
btw, small note on this, we should really start using, wherever possible, c.EVENT_NAME_AND_YEAR which would say 'Magfest 2016' instead of 'Magfest'.

This still might not have prevented the specific case this bug is talking about, the only other thing I can think of doing is putting the dates in the subject line like c.EPOCH -> c.ESCHATON.  Or perhaps have some kind of: 

```python
c.VOLUNTEER_EMAIL_SUBJECT = c.EVENT_NAME_AND_YEAR + " " + [event dates here]
```

It is dangerous to change the subject lines of emails right now since it will cause them to be re-sent.  might not be a bad idea in future events though.

**Comment by binary1230** [comment]
If you don't mind I think I'll keep this one open as an intern task, though we won't be able to work on it until a lull between events, since we judge whether we sent an email out by the exact text of the subject line.

**Comment by binary1230** [comment]
for everyone's reference, here's the current code that sends these out:
```python
StopsEmail('{EVENT_NAME} Volunteer Badge Confirmation', 'placeholders/volunteer.txt',
```

**Comment by binary1230** [comment]
OH HEY WAIT. if we did want to put other things in the subject line, we can't use ```c.WHATEVER``` here, we have to change ```AutomatedEmail.__init__()``` to pass in the new param here:
```self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)```

**Comment by kitsuta** [comment]
Oh, I do like the idea of putting the event dates in the subject line.

Since changing the subject of emails does resend them, someone could submit a PR to upstream for now.

**Comment by binary1230** [comment]
yea something like "Magfest 2/18/16" or "Magfest Feb 18-21" or "Magfest (February)" would do it

**Comment by binary1230** [comment]
relevant to trying to address the pain of renaming emails and not have them be re-sent: https://github.com/magfest/ubersystem/issues/1594

**Comment by bds002** [comment]
Before this get's merged, someone please make sure we don't resend all the emails in Rams/Uber for MAGFest 2016......

**Comment by binary1230** [comment]
nah it won't, it would only resend if we changed the subject line, #1610 only changes the body of the email, so it's cool.

**Comment by kitsuta** [comment]
I'm marking this as fixed from #1610.
